config/config.json

```diff
@@ -33,5 +33,6 @@
   "VAULT_CACERT": "config/myCA.crt",
   "VAULT_MTLS_CLIENT_CERT": "config/mtls_auth.crt",
   "VAULT_MTLS_CLIENT_KEY": "config/mtls_auth.key",
-  "VAULT_URL": "https://vault:8300"
+  "VAULT_URL": "https://vault:8300",
+  "VAULT_NAMESPACE": "tenant"
```

Comment:
Why do you put the namespace mapping into the general section of the config? The Vault namespace can vary per distributey tenant I think. Or am I missing something? If true, just move this config the `tenantX` under `TENANT_CFG`.

Reply:
I just added it there to have a minimal diff :)

You are right, if the distributey broker accesses several key Vaults (sources), all `VAULT_` related config keys should probably be added to the "distributey tenant" specific sections under `TENANT_CFG`, correct?

Example:
```
{
  "TENANT_CFG": {
    "monitoring": {
      "auth": {
      ...
      },
      "backend": {
      ...
      },
      "vault_auth_jwt_path": "jwt-monitoring",
      "vault_default_role": "monitoring",
      "vault_transit_path": "transit-monitoring",
      "vault_addr": "https://vault:8300",               <- VAULT_ADDR for "monitoring" distributey tenant
      "vault_namespace": "tenant",                      <- VAULT_NAMESPACE for "monitoring" distributey tenant
      "vault_cacert": "config/myCA.crt",                <- moved from global VAULT_CACERT 
      "vault_mtls_client_cert": "config/mtls_auth.crt", <- moved from global VAULT_MTLS_CLIENT_CERT 
      "vault_mtls_client_key": "config/mtls_auth.key",  <- moved from global VAULT_MTLS_CLIENT_KEY 
    },
    "consumer1": {},
    "consumer2": {},
      ...
  },
  <- no more global config keys here
}
```

Reply:
You're right, why not support multiple Vaults instead of namespaces. I think we can do both in one go!
Will work on this on the weekend :)